examples/proxy/main.tf

```diff
@@ -0,0 +1,150 @@
+variable "gcp_creds" {
+  default = ""
+  description = "base64 encoded json GCP key file for a service account"
```

Comment:
So this is the file or the content? I guess it can be both per https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference#authentication-configuration

Reply:
Basically you take the content of the file and you run it through base64. This is the cleanest way to pass it into GCP that supports both local use and CI based process